stdlib/2.7/sys.pyi

```diff
@@ -86,6 +86,9 @@ path_hooks = ...  # type: List[Any]
 path_importer_cache = ...  # type: Dict[str, Any]
 displayhook = ...  # type: Optional[Callable[[int], None]]
 excepthook = ...  # type: Optional[Callable[[type, BaseException, TracebackType], None]]
+exc_type = ...  # type: type
+exc_value = ...  # type: BaseException
```

Comment:
pytype models the difference between old-style and new-style. We're not getting a whole lot of value out of that, though, so we might drop it at some point.

- `sys.exc_type`: `Union[None, type, types.ClassType]` sounds good to me.
- `sys.exc_value`: `Any` (Because you can do `raise IOError, 42`)
- `sys.exc_traceback`: `TracebackType` (Without Optional.)

About `exc_traceback`: I can't seem to make this None. It either doesn't exist or it contains a traceback, even when doing things like 
```
raise IOError, IOError(), None
```
Am I missing something?